src/radical/pilot/pmgr/launching/default.py

```diff
@@ -427,6 +432,10 @@ def _start_pilot_bulk(self, resource, schema, pilots):
         rcfg = self._session.get_resource_config(resource, schema)
         sid  = self._session.uid
 
+        # we create a deep-copy of the resource cfg, so that this code is fee to
+        # change it as needed
+        rcfg = ru.Config(cfg=rcfg.as_dict())
```

Comment:
this one could be skipped, since `self._session.get_resource_config` already returns a deepcopy of config

Reply:
Indeed - fixed.